test/algorithms/repgradelbo.jl

```diff
@@ -27,6 +27,19 @@
         end
     end
 
+    @testset "estimate_objective" begin
+        @testset for alg in [KLMinRepGradDescent(AD), KLMinRepGradProxDescent(AD)]
+            obj_est = estimate_objective(rng, alg, q0_trans, model)
+            isfinite(obj_est)
+
+            obj_est = estimate_objective(rng, alg, q0_trans, model; n_samples=1)
+            isfinite(obj_est)
+
+            obj_est = estimate_objective(rng, alg, q0_trans, model; n_samples=3)
+            isfinite(obj_est)
```

Comment:
Could this check the value in some meaningful way? Either something for which the result is analytically known, or checking it against some other function that computes the same thing and is more extensively tested?

Reply:
Done!